i8086.cxx

```diff
@@ -1718,7 +1724,7 @@ uint64_t i8086::emulate( uint64_t maxcycles )
                     uint8_t tempal = al();
                     set_ah( tempal / _b1 );
                     set_al( tempal % _b1 );
-                    set_PSZ16( ax );
+                    set_PSZ8( al() );

```

Comment:
Intel's manual specifies AL instead of AX, and also has been tested on hardware via the aforementioned tests.